.changeset/fifty-rats-jog.md

```diff
@@ -0,0 +1,8 @@
+---
+"@pnpm/config": minor
```

Comment:
Is `minor` the right semver change for a flagged feature? This didn't feel like a `patch`.